### PR TITLE
deploy

### PR DESCRIPTION
**Pull request by rrudol**

### CODE REVIEW

File: public/data.json [hunk 1]
```diff
@@ -5,18 +5,24 @@
...
     "company": "Cognifide",
     "location": "Poznań",
     "image": "https://scontent-waw1-1.xx.fbcdn.net/v/t1.0-9/12289639_956454094411220_6576048838492494916_n.jpg?_nc_cat=106&_nc_ht=scontent-waw1-1.xx&oh=4c1ac470853b31696f62eb0e53ae7001&oe=5CC09EA2",
-    "salary": "6000 - 9500 PLN",
-    "level": "junior",
-    "type": "backend"
+    "salary": {
+      "min": "6000",
+      "max": "9500"
+    },
```
Comment (rrudol) [inline public/data.json]: what if only one value will be provided?

File: public/offer/2.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{
+  "id": "2",
+  "name": "Intern Software Developer",
+  "company": "Cognifide",
+  "location": "Poznań",
+  "image": "https://scontent-waw1-1.xx.fbcdn.net/v/t1.0-9/12289639_956454094411220_6576048838492494916_n.jpg?_nc_cat=106&_nc_ht=scontent-waw1-1.xx&oh=4c1ac470853b31696f62eb0e53ae7001&oe=5CC09EA2",
+  "salary": "3000 - 4500 PLN",
```
Comment (rrudol) [inline public/offer/2.json]: change to object as in 1.json

File: public/data.json (hunk 1 above)
Comment (rrudol) [reply public/data.json]: oh, I see 👍

File: public/data.json [hunk 3]
```diff
@@ -5,18 +5,24 @@
     "company": "Cognifide",
     "location": "Poznań",
     "image": "https://scontent-waw1-1.xx.fbcdn.net/v/t1.0-9/12289639_956454094411220_6576048838492494916_n.jpg?_nc_cat=106&_nc_ht=scontent-waw1-1.xx&oh=4c1ac470853b31696f62eb0e53ae7001&oe=5CC09EA2",
-    "salary": "3000 - 4500 PLN",
-    "level": "internship",
-    "type": "backend"
+    "salary": {
+      "min": "",
+      "max": ""
+    },
```
[the remark this answers is not in the document]
Comment (mateusz-stempniewicz) [reply public/data.json]: if both values are empty then component show "Bezpłatnie"

File: public/offer/2.json (hunk 2 above)
Comment (mateusz-stempniewicz) [reply public/offer/2.json]: ok :+1:

**Pull request merged**